base_geoengine/geo_polygon.py

```diff
@@ -18,17 +18,18 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
-from __future__ import absolute_import
-from osv import fields
+from openerp.osv import fields
 from . import geo_field
 
+
 class GeoPolygon(geo_field.Geom):
     """New type of column in the  ORM for POSTGIS geometry Polygon type"""
     _type = 'geo_polygon'
+    _geo_type = 'POLYGON'
 
-    def __init__(self, string, dim=2, srid=900913 , gist_index=True, **args):
-        res = super(GeoPolygon, self).__init__(string, "POLYGON",
-                                               dim=dim, srid=srid, gist_index=gist_index, **args)
-        return res
+    def __init__(self, string, dim=2, srid=900913, gist_index=True, **args):
```

Comment:
Inheriting the docstring from super can help document what the parameters are supposed to be.